superset-frontend/src/features/tags/TagModal.tsx

```diff
@@ -234,10 +238,14 @@ const TagModal: React.FC<TagModalProps> = ({
           name: tagName,
           objects_to_tag: [...dashboards, ...charts, ...savedQueries],
         },
-      }).then(({ json = {} }) => {
-        refreshData();
-        addSuccessToast(t('Tag created'));
-      });
+      })
+        .then(({ json = {} }) => {
+          refreshData();
+          addSuccessToast(t('Tag created'));
+        })
+        .catch(err =>
+          addDangerToast(err.message ? err.message : 'Error Creating Tag'),
```

Comment:
```suggestion
          addDangerToast(err.message || 'Error Creating Tag'),
```

Reply:
maybe also translate the error and you can define it elsewhere with the translation since it's used in multiple places.